website/content/docs/upgrading/upgrade-specific.mdx

```diff
@@ -21,6 +21,13 @@ After Consul 1.17.0, only timestamps will be included in rotated log file names.
 
 ## Consul 1.16.x
 
+#### Vault Enterprise as CA
+Using Vault as CA with Consul version 1.16.2 will fail to initialize the CA if [`namespace`](/consul/docs/connect/ca/vault#namespace) is set
+but [`intermediate_pki_namespace`](/consul/docs/connect/ca/vault#intermediatepkinamespace) or [`root_pki_namespace`](/consul/docs/connect/ca/vault#rootpkinamespace)
+are empty. This is a bug which will be fixed in a future version.
+To work around this issue, users must explicitly set [`intermediate_pki_namespace`](/consul/docs/connect/ca/vault#intermediatepkinamespace) and
```

Comment:
Are we concerned someone might think it's sufficient to just change the CA config in the agent config, rather than actually using the API/CLI to apply the change?

Maybe we could recommend reading back the config from the CLI / API to ensure the intended change has taken effect?

Reply:
Added steps to set-config then get-config